.travis.yml

```diff
@@ -1,24 +1,20 @@
 language: go
 
 go:
-  - "1.8"
-  - "1.9"
-  - "1.10"
   - "1.11"
+  - "1.12"
   - master
 
 env:
   global:
     - REVIEW_DOG_ON=1.11
     - REVIEW_DOG_VERSION=0.9.11
-    - VERSION_DIR=v2
```

Comment:
`- GO111MODULE=on` is needed for 1.11 and 1.12.
`on` will be default on 1.13 (master).